pkgdb/tests/environment.cc

```diff
@@ -902,6 +920,104 @@ test_createLockfile_error()
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+/**
+ * @brief `getCombinedRegistryRaw()` uses the locked input and doesn't lock the
+ *        unlocked input in an environment's manifest.
+ *
+ * Re-locking the locked input causes a useless download of nixpkgs.
+ */
+bool
+test_getCombinedRegistryRaw_uses_lock()
+{
+  /* Create manifest with a registry with an invalid nixpkgs ref */
```

Comment:
suggestion: add some motivation for "invalid nixpkgs"

i guess you would expect locking to throw an error if was using this nixpkgs right, and the test succeeds because it isnt..
I dont think that's inherently clear